mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule Monarch.MixProject do
   def project do
     [
       app: :monarch,
-      version: "0.1.5",
+      version: "0.1.6",
       elixir: "~> 1.17.2",
```

Comment:
Update the elixir version?? 

![Screenshot 2025-03-26 at 8 43 07 AM](https://github.com/user-attachments/assets/bb369819-2a83-4951-9f6b-f7353cd14db0)